lesson_1.1/src/TodoListItem.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
-function TodoListItem({todo}) {
+function TodoListItem({todo,onRemoveTodo}) {
     console.log({todo})
     return (
         <div>
-           <li key={todo.id}>{todo.title}</li> 
+           <li key={todo.id}>{todo.title}<button  onClick={()=>onRemoveTodo(todo.id)} type="button">Remove</button> </li> 
```

Comment:
this line is starting to get a bit long and unreadable. you'd benefit from breaking up the tags into a different line, the contents inside the list tags into a different line:

list w/ key
{todo.title}
button w/ props/attributes 
"Remove"
button closing tag
list closing tag

(with proper indentation since comments aren't letting me do it)